Common/include/option_structure.hpp

```diff
@@ -371,13 +371,17 @@ enum ENUM_KIND_NONDIM {
   DIMENSIONAL = 0,			    /*!< \brief Dimensional simulation. */
   FREESTREAM_PRESS_EQ_ONE = 1, /*!< \brief Non-dimensional simulation. */
   FREESTREAM_VEL_EQ_MACH = 2, /*!< \brief Non-dimensional simulation. */
-  FREESTREAM_VEL_EQ_ONE = 3 /*!< \brief Non-dimensional simulation. */
+  FREESTREAM_VEL_EQ_ONE = 3,
```

Comment:
I know this is how we've always done it, but enums don't need to have explicit values set unless there's somewhere that it's converted to an int for a function that doesn't have access to the enum definition. Should probably leave it this way, since it's consistent with the rest of the code.

Reply:
Maybe we should remove this everywhere. Often this leads to unnecessary conflicts.